### ISSUE TITLE
Scanners and transient

### ISSUE THREAD

**Issue by KJHeeres** [rug-oop/2019_assignments#16]
When we make our scanners transient they start throwing java.lang.NullPointerException after we save and load the rpg. However, when we make them static they don't do this. We were wondering whether there is a way of making this work without making the scanners static.

**Comment by ghidirimschi** [comment]
Why do you need a scanner instance for every object that you create in the first place? All of them use the same source - System.in.

**Comment by KJHeeres** [comment]
Fair point, thanks for the quick response. Would it then  be best to make a single instance of scanner in main, and use that one for everything?
